### PR TITLE
Balance pass two

### PR DESCRIPTION
**Pull request by ProbablyCarl**
- - -
Balance:
 - Assault Carbines nerfed in damage and spread. Significantly. This effects standard and worn.
 - AMR now requires the player to be scoped in, or, with a 95% chance, they'll be stunlocked(Knocked to the floor, capable of moving and fighting still. Just far slower.) for firing unprepared.
 - NCR's SEA given a worn assault carbine, rather than standard, identically to the Pointman loadout of Sergeant.
- - -
New:
 - Outcast and Remnant loadouts added to Outlaw. Minor 'unique' stuff for the Remnant has been provided, in both two gimped plasma grenades, and a very meh plasma pistol.
 - Brotherhood Exile role moved to Wastelander.
 - Stuff.
- - -
QoL:
 - Minigun no longer generates casings.
 - Enclave provided with syndicate IDs for espionage purposes.
 - Wiki button renamed from 'TG Wiki' to simply 'Wiki', command updated accordingly.
- - -
Why this includes the pixel shift stuff?
No idea. I swear, sometimes Tortoise Git is wacky like that.
- - -

### CODE REVIEW

File: code/game/objects/items/granters.dm [hunk 1]
```diff
@@ -834,6 +834,11 @@
 	icon_state = "blueprint2"
 	crafting_recipe_types = list(/datum/crafting_recipe/explosive/shrapnelmine, /datum/crafting_recipe/explosive/explomine, /datum/crafting_recipe/explosive/stingmine, /datum/crafting_recipe/explosive/pulsemine, /datum/crafting_recipe/explosive/stunmine)
 
+/obj/item/book/granter/crafting_recipe/blueprint/plasmapistol_rem
+	name = "plasma sidearm repair guide"
```
Comment (Spc-Dragonfruits) [inline code/game/objects/items/granters.dm:838]:
```suggestion
	name = "plasma pistol field blueprint"
```
For clarity's sake

File: code/game/objects/items/grenades/f13grenade.dm [hunk 2]
```diff
@@ -54,6 +54,13 @@
 	shrapnel_radius = 10
 	var/rad_damage = 300
 
+/obj/item/grenade/f13/plasma/remnant
+	name = "dusty plasma grenade"
```
Comment (Spc-Dragonfruits) [inline code/game/objects/items/grenades/f13grenade.dm:58]:
```suggestion
	name = "expired plasma grenade"
```

File: code/modules/projectiles/guns/energy/plasmaf13.dm [hunk 3]
```diff
@@ -32,6 +32,12 @@
 	ammo_type = list(/obj/item/ammo_casing/energy/plasma/pistol/worn)
 	fire_delay = 4.5
 
+/obj/item/gun/energy/laser/plasma/pistol/remnant
+	name ="lightweight plasma pistol"
```
Comment (Spc-Dragonfruits) [inline code/modules/projectiles/guns/energy/plasmaf13.dm:36]:
```suggestion
	name ="corroded plasma pistol"
```
Clarity's sake, since we got another one

**Review by Spc-Dragonfruits — CHANGES_REQUESTED** [review]
Mostly QOL stuff

**Review by PolymorphBlue — APPROVED** [review]
Looks fine.

**Pull request merged**